assets/js/script.js

```diff
@@ -0,0 +1,31 @@
+var mediaSelectEl = document.getElementById("media-select");
+var searchInputEl = document.getElementById("search-input");
+var submitButtonEl = document.getElementById("submit-button");
+
+// funtion to check which media types are selected
+// then send input to correct fetch function
+var formHandler = function (event) {
+    event.preventDefault();
+    // define user input from form
+    var searchTerm = searchInputEl.value;
+    // define type of media user selected
+    var selectedMedia = mediaSelectEl.value;
+    // send user input to appropriate fetch function
+    if (selectedMedia === "movies") {
+        console.log("sent to movies");
+        // send searchTerm to Mason's movie fetch function
+        // movieFetchHandler(searchTerm);
+    }
+    if (selectedMedia === "music") {
+        console.log("sent to music");
+        // send searchTerm to music fetch function
+        // musicFetchHandler(searchTerm);
+    }
+    else if (selectedMedia === "books") {
+        console.log("sent to books");
+        // send searchTerm to book fetch function
+        // bookFetchHandler(searchTerm);
+    }
```

Comment:
I think it would be good to make some nested conditionals inside these conditionals which look for the value of the second select box and then branch out to the appropriate fetch request. Of course, we don't know what values we are including in our search yet, but it would be good to have a basic structure set up.

Reply:
I thought it would be easiest to put those conditionals in the following fetch request functions and adjust the request there, rather than send to a different fetch entirely depending on that second select. I could be misunderstanding though, what do you think about that?

Reply:
You may be right. Lets go ahead with that idea for now.